### ISSUE TITLE
CMD shortcut keys required main window to be active

### ISSUE THREAD

**Issue by TheTechnobear** [Eigenlabs/EigenD#15]
Shortcut keys (cmd+1 etc) appear to be broken (mavericks?), every now and again they work for me, but not all the time (it may be closing window related, not sure)

EDIT: appears the issue is already present on 2.0 and 10.8, the issue is main window has to be active, rather than the application (which is the UI norm).

**Comment by keymanpal** [comment]
macmini -mavericks-2.0.74
It works for me ok… EigenD have always to have the focus; for shortcuts to work… aaaa who am I to say this ;-)

**Comment by GoneCaving** [comment]
The fact that the EigenD window had to have focus for the shortcuts to work
always annoyed me. I wonder how hard that would be to fix?

**Comment by TheTechnobear** [comment]
Possibly 2.1 only then.
Can you try closing the windows, also ensure you have multiple windows
open. Just different combinations.
Also do you have more than 9 windows, ie some don't have shortcuts
It's doing most of the time for me, but not 100%
I will check factory setups do it.

Thanks
Mark

On 07/05/2014, at 20:16, keymanpal notifications@github.com wrote:

macmini -mavericks-2.0.74
It works for me ok… EigenD have always to have the focus; for shortcuts to
work… aaaa who am I to say this ;-)

—
Reply to this email directly or view it on
GitHubhttps://github.com/Eigenlabs/EigenD/issues/15#issuecomment-42462990
.

**Comment by TheTechnobear** [comment]
Are you saying the 'load' window has to have focus?  I'm assuming it's the
application (ie it's the foreground app)
Foreground app is normal , particular window is not :)

On 07/05/2014, at 20:21, Duncan Foster notifications@github.com wrote:

The fact that the EigenD window had to have focus for the shortcuts to work
always annoyed me. I wonder how hard that would be to fix?

On Wed, May 7, 2014 at 3:16 PM, keymanpal notifications@github.com wrote:

> macmini -mavericks-2.0.74
> It works for me ok… EigenD have always to have the focus; for shortcuts
> to
> work… aaaa who am I to say this ;-)
> 
> —
> Reply to this email directly or view it on GitHub<
> https://github.com/Eigenlabs/EigenD/issues/15#issuecomment-42462990>
> .

—
Reply to this email directly or view it on
GitHubhttps://github.com/Eigenlabs/EigenD/issues/15#issuecomment-42463696
.

**Comment by keymanpal** [comment]
yep… add me to the "annoyed" also and yes it has to be the load window… bummer.

**Comment by TheTechnobear** [comment]
cool, that at least makes it 'reproducible' , but its still a bug :)

the only justification i see for this behaviour, is if its been made windows specific because plugins may already be using this shortcut... this would cause a problem.
are we aware of a plugin that uses these shortcuts in windows or mac?

if this is the case, i still see as a bug, but we will need user customisable shortcuts to fix (so you can change away from a clash with the plugin) , this would therefore have to be solved alongside issue #16 .
